src/applications/personalization/profile360/components/EmailEditModal.jsx

```diff
@@ -14,7 +14,7 @@ export default class EditEmailModal extends React.Component {
       return { ...this.props.data };
     }
     return {
-      email: ''
+      emailAddress: ''
```

Comment:
This is the correct value based on the object we need to set to Vet360, but the name in the modal reducer is `email`; down the line we may want to make these consistent for clarity

Reply:
Yeah for sure. Any way we can consolidate the terminology sounds great to me. 

This is a tricky one because `email` comes from the profile's Vet360 contact information, and `email.emailAddress` is how the actual value is accessed. Confusing.